extension/src/shared/ui-components/MyData/index.tsx

```diff
@@ -0,0 +1,50 @@
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import TabPanel from "@mui/lab/TabPanel";
+import Box from "@mui/material/Box";
+import Tab from "@mui/material/Tab";
+import { useAtom } from "jotai";
+import { useCallback, useState } from "react";
+
+import { useAddLayer } from "../../../prototypes/layers";
+import { showMyDataModalAtom } from "../../../prototypes/view/states/app";
+import SharedModal from "../Modal";
+
+import LocalDataTab from "./LocalDataTab";
+import WebDataTab from "./WebDataTab";
+
+const MyData = () => {
```

Comment:
Could you move this component to `/shared/ui-container/MyData.tsx` like `Feedback` component? Because I would like to keep `ui-components` to define just UI.